package.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/bash -ex
-export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11.5'
+export DEBIFY_IMAGE='registry.tld/conjurinc/debify:2.0.0'
 
 docker run --rm $DEBIFY_IMAGE config script > docker-debify
 chmod +x docker-debify
 
+# Create possum deb
 ./docker-debify package \
   --dockerfile=Dockerfile.fpm \
+  --output=deb \
+  possum \
+  -- \
+  --depends tzdata \
+  --rpm-rpmbuild-define '_build_id_links none'
```

Comment:
what's this flag? it is also failing the build

Reply:
Forgot in rpm linesep it failed the build